pipelines/snap-atac/README.md

```diff
@@ -1,28 +1,33 @@
 # Snap-ATAC pipeline Overview
 
-The snap-atac pipeline is a pipeline for processing scATAC-seq datasets, based on the pipeline that is available [in the snap-atac repository](https://github.com/r3fang/SnapATAC] and (the snap-utils repository)[https://github.com/r3fang/SnapATAC).
+The snap-atac pipeline is a pipeline for processing scATAC-seq datasets, based on the pipeline that is available (in the snap-atac repository)[https://github.com/r3fang/SnapATAC) and [the snap-utils repository](https://github.com/r3fang/SnapATAC).
```

Comment:
This is not formatted correctly. Should be [in the snap-atac repository](https://github.com/r3fang/SnapATAC) and [the snap-utils repository](https://github.com/r3fang/SnapATAC)

Reply:
It formatted my comment... it should be [] around  'in the snap-atac repository' and () around the url